susemanager-utils/susemanager-sls/salt/transactional_reboot/config.sls

```diff
@@ -0,0 +1,16 @@
+copy_conf_file_to_etc_if_not_there:
```

Comment:
Do you already plan to add more `.sls` files into `transactional_reboot/` directory? Otherwise, you can think about replacing this directory with a single file named something like `transactional_reboot.sls` like the others.

Reply:
IMHO even if there is only this SLS file, I would to not place single SLS files in the main Salt "file_root" directory, but always use directories instead, so we can keep SLS files more organized.